src/frontend/context/EøsContext.test.tsx

```diff
@@ -14,58 +12,21 @@ import {
     mekkGyldigSøker,
     mekkGyldigUtenlandsoppholdEøs,
     mekkGyldigUtenlandsoppholdIkkeEøs,
+    TestProvidere,
 } from '../utils/testing';
 
-import { AppProvider } from './AppContext';
-import { preferredAxios } from './axios';
 import { EøsProvider, useEøsContext } from './EøsContext';
-import { InnloggetProvider } from './InnloggetContext';
-import { LastRessurserProvider } from './LastRessurserContext';
 import * as pdlRequest from './pdl';
-import { SanityProvider } from './SanityContext';
-import { SpråkProvider } from './SpråkContext';
 
 describe('EøsContext', () => {
-    const forkortetListeAvEøsLand = {
-        BEL: 'Belgia',
-        DNK: 'Danmark',
-        CHE: 'Sveits',
-        IRL: 'Irland',
-        NLD: 'Nederland',
-    };
-
     const wrapper = ({ children }) => (
-        <CookiesProvider>
-            <SpråkProvider>
-                <LastRessurserProvider>
-                    <InnloggetProvider>
-                        <SanityProvider>
-                            <IntlProvider locale="no">
-                                <AppProvider>
-                                    <EøsProvider>{children}</EøsProvider>
-                                </AppProvider>
-                            </IntlProvider>
-                        </SanityProvider>
-                    </InnloggetProvider>
-                </LastRessurserProvider>
-            </SpråkProvider>
-        </CookiesProvider>
+        <TestProvidere>
+            <EøsProvider>{children}</EøsProvider>
+        </TestProvidere>
     );
 
     beforeEach(() => {
-        const axiosMock = new MockAdapter(preferredAxios);
-
-        axiosMock.onGet(/\/api\/innlogget/).reply(200, {
-            status: RessursStatus.SUKSESS,
-            data: 'Autentisert kall',
-        });
-
-        axiosMock.onGet(/\/kodeverk\/eos-land/).reply(200, {
-            status: RessursStatus.SUKSESS,
-            data: forkortetListeAvEøsLand,
-        });
-
-        jest.spyOn(pdlRequest, 'hentSluttbrukerFraPdl').mockImplementation(async () => ({
+        vi.spyOn(pdlRequest, 'hentSluttbrukerFraPdl').mockImplementation(async () => ({
```

Comment:
Kunne ha jobbet mer for å fjerne denne mocken, men det er et eller annet funky som gjør at den ikke lar seg fjerne. Lar den være enn så lenge, et sted må denne PRen være ferdig. 😅